spec/apollo-federation/service_field_v1_spec.rb

```diff
@@ -455,6 +455,30 @@ def self.visible?(context)
       )
     end
 
+    it 'returns SDL that inherits object directives' do
+      base_object_with_id = Class.new(base_object) do
+        key fields: 'id'
+
+        field :id, String, null: false
+      end
+
+      product = Class.new(base_object_with_id) do
+        graphql_name 'Product'
+      end
+
+      schema = Class.new(base_schema) do
+        orphan_types product
+      end
+
+      expect(execute_sdl(schema)).to match_sdl(
+        <<~GRAPHQL,
+          type Product @key(fields: "id") {
+            id: String!
+          }
+        GRAPHQL
+      )
+    end
```

Comment:
Do you want to add a schema inheritance test here as well?

Reply:
The [v1](https://www.apollographql.com/docs/federation/v1/federation-spec/#federation-schema-specification) federation spec doesn't support any schema level directives so I don't think there's anything to test here.